### PR TITLE
[zh-cn]: Update translation of Object.create()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Such as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

None.

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/create/index.md [hunk 1]
```diff
@@ -21,106 +21,16 @@ Object.create(proto, propertiesObject)
 - `proto`
   - : 新创建对象的原型对象。
 - `propertiesObject` {{Optional_inline}}
-  - : 如果该参数被指定且不为 {{jsxref("undefined")}}，则该传入对象的自有可枚举属性（即其自身定义的属性，而不是其原型链上的枚举属性）将为新创建的对象添加指定的属性值和对应的属性描述符。这些属性对应于 {{jsxref("Object.defineProperties()")}} 的第二个参数。
+  - : 如果该参数被指定且不为 {{jsxref("undefined")}}，则该传入对象[可枚举的自有属性](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)将为新创建的对象添加指定的属性值和对应的属性描述符。这些属性对应于 {{jsxref("Object.defineProperties()")}} 的第二个参数。
 
 ### 返回值
 
 一个新对象，带着指定的原型对象及其属性。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/create/index.md:28]:
```suggestion
根据指定的原型对象和属性创建的新对象。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/create/index.md [hunk 2]
```diff
@@ -21,106 +21,16 @@ Object.create(proto, propertiesObject)
...
-// A malicious script:
-Object.prototype.authenticated = true;
-
-// Unexpectedly allowing unauthenticated user to pass through
-if (user.authenticated) {
-  // access confidential data...
-}
-```
+- {{jsxref("TypeError")}}
+  - : 如果 `proto` 既不是 {{jsxref("null")}}，也不是一个 {{jsxref("Object")}}，则抛出此错误。
```
Comment (fwqaaq) [inline files/zh-cn/web/javascript/reference/global_objects/object/create/index.md:33]:
```suggestion
  - : 如果 `proto` 既不是 {{jsxref("null")}}，也不是 {{jsxref("Object")}}，则抛出此错误。
```

前半句没有量词，后半句也去掉量词

File: files/zh-cn/web/javascript/reference/global_objects/object/create/index.md [hunk 3]
```diff
@@ -129,93 +39,91 @@ if (user.authenticated) {
...
-// subclass extends superclass
-Rectangle.prototype = Object.create(Shape.prototype);
-
-//If you don't set Rectangle.prototype.constructor to Rectangle,
-//it will take the prototype.constructor of Shape (parent).
-//To avoid that, we set the prototype.constructor to Rectangle (child).
-Rectangle.prototype.constructor = Rectangle;
+// 子类继承父类
+Rectangle.prototype = Object.create(Shape.prototype, {
+  // 如果不将 Rectangle.prototype.constructor 设置为 Rectangle，
```
[the remark this answers is not in the document]
Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/object/create/index.md:62]: 合并后会产生换行注释，观感不佳。一般合并后能控制在80个字符内的我会合并的

File: files/zh-cn/web/javascript/reference/global_objects/object/create/index.md [hunk 4]
```diff
@@ -21,106 +21,16 @@ Object.create(proto, propertiesObject)
 - `proto`
   - : 新创建对象的原型对象。
 - `propertiesObject` {{Optional_inline}}
-  - : 如果该参数被指定且不为 {{jsxref("undefined")}}，则该传入对象的自有可枚举属性（即其自身定义的属性，而不是其原型链上的枚举属性）将为新创建的对象添加指定的属性值和对应的属性描述符。这些属性对应于 {{jsxref("Object.defineProperties()")}} 的第二个参数。
+  - : 如果该参数被指定且不为 {{jsxref("undefined")}}，则该传入对象[可枚举的自有属性](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)将为新创建的对象添加指定的属性值和对应的属性描述符。这些属性对应于 {{jsxref("Object.defineProperties()")}} 的第二个参数。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/create/index.md:24]:
```suggestion
  - : 如果该参数被指定且不为 {{jsxref("undefined")}}，则该传入对象[可枚举的自有属性](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)将为新创建的对象添加具有对应属性名称的属性描述符。这些属性对应于 {{jsxref("Object.defineProperties()")}} 的第二个参数。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/create/index.md [hunk 5]
```diff
@@ -129,93 +39,91 @@ if (user.authenticated) {
...
-console.log('Is rect an instance of Rectangle?', rect instanceof Rectangle); // true
-console.log('Is rect an instance of Shape?', rect instanceof Shape); // true
-rect.move(1, 1); // Outputs, 'Shape moved.'
+console.log("rect 是 Rectangle 类的实例吗？", rect instanceof Rectangle); // true
+console.log("rect 是 Shape 类的实例吗？", rect instanceof Shape); // true
+rect.move(1, 1); // 打印 'Shape moved.'
 ```
 
-### 使用 `Object.create()` 的 `propertyObject` 参数
+需要注意的是，使用 `create()` 也有一些要注意的地方，比如重新添加 [`constructor`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor) 属性以确保正确的语义。虽然 `Object.create()` 被认为比使用 {{jsxref("Object.setPrototypeOf()")}} 修改原型更具有性能优势，但如果没有创建实例并且属性访问还没有被优化，它们之间的差异实际上是可以忽略不计的。在最佳实践中，无论如何都应该优先使用[类](/zh-CN/docs/Web/JavaScript/Reference/Classes)。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/create/index.md:80]:
```suggestion
需要注意的是，使用 `create()` 也有一些要注意的地方，比如重新添加 [`constructor`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor) 属性以确保正确的语义。虽然 `Object.create()` 被认为比使用 {{jsxref("Object.setPrototypeOf()")}} 修改原型更具有性能优势，但如果没有创建实例并且属性访问还没有被优化，它们之间的差异实际上是可以忽略不计的。在现代代码中，无论如何都应该优先使用[类](/zh-CN/docs/Web/JavaScript/Reference/Classes)语法。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**